src/ape/api/providers.py

```diff
@@ -203,7 +205,7 @@ class ReceiptAPI(BaseInterfaceModel):
     receiver: str
     nonce: Optional[int] = None
 
-    def __repr__(self) -> str:
+    def __repr__(self):
```

Comment:
Was this causing an issue?

Reply:
No, but it is also is not working for some reason, I was trying to fix it.
I will put back the `-> str` though!